README.adoc

```diff
@@ -269,6 +274,78 @@ properties are automatically stored in the inventory. Go back to http://localhos
 you see a new entry for `[system-ip-address]`. 
 
 
+== Externalizing server configuration
+
+// File 0
+inventory/server.xml
+[source, xml, linenums, indent=0, role="code_column"]
+----
+include::finish/inventory/src/main/liberty/config/server.xml[]
+----
+
+As mentioned in the beginning of this guide, one of the advantages of using
```

Comment:
Change,
"As mentioned in the beginning of this guide"
to,
"As mentioned at the beginning of this guide"